src/styles.css

```diff
@@ -304,18 +287,35 @@ button:hover {
   display: none
 }
 
+@media screen and (max-width: 425px) {
```

Comment:
I think this media query needs to be below the 768 max width one so they adjust one at a time. Not positive though

Reply:
I think you are correct I moved it and pushed up the changes.